.github/workflows/prs_check-if-pr-has-type-label.yml

```diff
@@ -3,15 +3,15 @@ name: Check PR for type labels
 on:
   workflow_call:
 
-permissions: {}
+permissions:
+  issues: write
+  pull-requests: write
+  contents: write
```

Comment:
I think that this is a strict no-go from Olivier, because of security auditing. 🤔 
Are you sure it is necessary? 🤔 
All the other workflows are fine with permissions defined on the `job` level. 🤷

Reply:
i know ... this should be on job level ... just wanted to try this instead to see if this was the culprit before ... but i guess it does not make a difference.